static/app/components/events/breadcrumbs/breadcrumbsDrawerContent.tsx

```diff
@@ -41,27 +35,25 @@ export const enum BreadcrumbControlOptions {
 
 interface BreadcrumbsDrawerContentProps {
   breadcrumbs: EnhancedCrumb[];
-  event: Event;
-  group: Group;
-  project: Project;
   focusControl?: BreadcrumbControlOptions;
 }
 
 export function BreadcrumbsDrawerContent({
-  event,
-  group,
-  project,
   breadcrumbs,
-  focusControl,
+  focusControl: initialFocusControl,
 }: BreadcrumbsDrawerContentProps) {
   const organization = useOrganization();
   const theme = useTheme();
+
   const [search, setSearch] = useState('');
   const [filterSet, setFilterSet] = useState(new Set<string>());
   const [sort, setSort] = useLocalStorageState<BreadcrumbSort>(
     BREADCRUMB_SORT_LOCALSTORAGE_KEY,
     BreadcrumbSort.NEWEST
   );
+
+  const [focusControl, setFocusControl] = useState(initialFocusControl);
```

Comment:
nit: This is a bit disjointed from the useCallback below, it might be nice to combine them into a hook like `useFocusControl()` which return `getFocusProps`

Reply:
That makes sense, I never thought of grouping hooks in this way, thanks!